pkg/config/config.go

```diff
@@ -1539,7 +1541,16 @@ func setupFipsEndpoints(config Config) error {
 	os.Unsetenv("HTTPS_PROXY")
 
 	// HTTP for now, will soon be updated to HTTPS
-	protocol := "http://"
+	var protocol string
+	if config.GetBool("fips.https") {
+		protocol = "https://"
+		// top-level setting takes precedence
+		if config.GetBool("skip_ssl_validation") {
+			config.Set("skip_ssl_validation", !config.GetBool("fips.tls_verify"))
```

Comment:
I'm wondering if we can just use `skip_ssl_validation` in the global setting instead of a dedicated flag in `fips` section here? Seem like `fips.tls_verify` won't serve much of a purpose right now if it requires for `skip_ssl_validation = true` to be used and even then it may be of marginal utility. Maybe we just need a flag for the frontend proxy certificate path instead?

Reply:
I thought about this; I do agree that `skip_ssl_validation` does the job and might be enough. I was lured into adding a new `tls_verify` option for the following reasons: 
- Because there are a few other places in the config file with dedicated `tls_verify` flags in  config subsections (eg. `kubelet_tls_verify` and `syslog_tls_verify`) 
- Both the options above deal with interactions with local entities (like the proxy), so customer might have muscle memory.
- it would also be less likely for customers to misconfigure if the options are self-contained.

That said, given how the code will always enforce the most secure combo and stuff such like `skip_ssl_validation==true` and `fips.tls_verify==false` will not enforce `fips.tls_verify` indeed the flag becomes less useful. I think it should come down to usability, ultimately.

Reply:
👍 As discussed we can implement the flag with precedence over `skip_ssl_validation`.